README.md

```diff
@@ -21,13 +21,21 @@ To install the plugin, run:
 
 ### Commands
 
-- `ceph`: Run a Ceph CLI command. Supports any arguments the `ceph` command supports. See [Ceph docs](https://docs.ceph.com/en/pacific/start/intro/) for more.
+- `ceph <args>` : Run a Ceph CLI command. Supports any arguments the `ceph` command supports. See [Ceph docs](https://docs.ceph.com/en/pacific/start/intro/) for more.
 
-- `operator`: run sub-commands against the Rook-Ceph operator
-  -  `restart`: restart the Rook-Ceph operator
+- `rbd <args>` : Call a 'rbd' CLI command with arbitrary args"
 
-- `help`: Output help text
+- `mon-endpoints` : print mon endpoints"
 
+- `operator <subcommands>` : run sub-commands against the Rook-Ceph operator mentioned below
```

Comment:
I think we don't need to mention the details, it seems obvious the subcommands are below, especially when they look at the later examples.
```suggestion
- `operator`
```